apps/docs/pages/guides/platform/migrating-and-upgrading-projects.mdx

```diff
@@ -12,6 +12,20 @@ In some cases, access to new features require upgrading or migrating your Supaba
 
 ## Upgrade your project
 
+### pg_upgrade
+
+<Admonition type="note">
+  This upgrade mechanism is currently in Beta, and is being slowly made available to projects.
+</Admonition>
```

Comment:
Adding newline at beginning & end of JSX tag prevents prettier from wrapping the content (if someone were to add to it).

```suggestion
<Admonition type="note">

This upgrade method is currently in Beta, and is being slowly made available to projects.

</Admonition>
```

Reply:
Not broken down along that dimension